lib/diagnostic.rb

```diff
@@ -69,18 +71,19 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'nothing will be returned as there is
+nothing asking to be returned'
 
```

Comment:
Correct, if this were javascript. Try running this code in pry and see what gets returned ;)